extension/persistence/relational-jdbc/src/main/resources/db.properties

```diff
@@ -0,0 +1,31 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+jdbc.url=jdbc:postgresql://localhost:5432/postgres
+jdbc.username=testuser
```

Comment:
Hmmm test data in a prod file?

Reply:
It expected that the user overrides the db.properties files the way it does for eclipselink and is kind of default for testing and quick dev perspective.

Reply:
I'm not sure the current way for configuring EclipseLink in Polaris is a good example :sweat_smile: 

How can a user override something that is compiled into a jar? If recompilation is required, I would say it's a "user", in that case it's a "developer", from my POV.

Reply:
I see, if this can be done via Quarkus easily, happy to try it out.
Do you have any specific way in mind ?

Reply:
commented in other threads with specific suggestions :)